examples/platform/silabs/SiWx917/SiWx917/hal/rsi_bootup_config.c

```diff
@@ -0,0 +1,369 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *
+ * you may not use this file except in compliance with the License.
```

Comment:
this license seems somehow mis-aligned. Could we update it?

Reply:
deleted this file since it was not used and is under the submodule